src/Components/Shared/Pagination.js

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Pagination, PaginationItem, PaginationLink } from "reactstrap";
+
+export default class DotPagination extends React.Component {
+  constructor(props, context) {
+    super(props, context);
+
+    this.state = {
+      activePage: 1,
+      pageSize: 0,
+      totalItems: 0
+    };
+  }
+
+  componentDidMount() {
+    this.setState(state => ({
+      ...state,
+      pageSize: this.props.sizePerPage,
```

Comment:
This is a anti pattern in react, init state with props: https://stackoverflow.com/a/47341539/3367318.

But also pageSize and totalItems doesn't change... you should be able to use the props directly.